server/routes/education-support-plan/validationSchemas/specificTeachingSkillsSchema.test.ts

```diff
@@ -54,7 +54,7 @@ describe('specificTeachingSkillsSchema', () => {
       const expectedErrors: Array<Error> = [
         {
           href: '#skillsRequired',
-          text: 'Select whether any specific teaching skills are required',
+          text: "Select if teachers need specific knowledge or skills to support person's learning",
```

Comment:
It's as per the ticket 👍 , but 
> Select if teachers need specific knowledge or skills to support person's learning
just reads weird to me. I would have thought it should be "... to support a person's learning" or "... the person's learning"

But meh 🤷‍♂️ , what do I know? 🤣  You've done as the ticket has asked for 👍